docs/apis-tools/zeebe-api-rest/zeebe-api-rest-authentication.md

```diff
@@ -15,7 +15,7 @@ Your client must send a header in each request:
 For example, send a request using _curl_:
 
 ```shell
-curl -X POST -H -H :accept: application/json" -H "Authorization: Bearer <TOKEN>" -d '' http://localhost:8080/v1/topology
+curl -XGET -H'Accept: application/json' -H'Authorization: Bearer <TOKEN>' http://localhost:8080/v1/topology
```

Comment:
🔧 I think we're missing a whitespace here, don't we?
```suggestion
curl -X GET -H'Accept: application/json' -H'Authorization: Bearer <TOKEN>' http://localhost:8080/v1/topology
```

Reply:
AFAIK it doesn't matter. Same with the `-H'Authorization'` for example :shrug: